app/src/main/kotlin/com/mapzen/erasermap/view/ViewRecycler.kt

```diff
@@ -0,0 +1,35 @@
+package com.mapzen.erasermap.view
+
+import android.view.View
+import java.util.HashMap
+import java.util.HashSet
+
+class ViewRecycler {
```

Comment:
This definitely is spartan but it allows recycling the views of the inner `LinearLayout` which doesn't have recycling built into it. There are two types of views that get recycled- `R.layout.instruction_row` and `R.layout.transit_station_row`